homeassistant/components/vesync/__init__.py

```diff
@@ -62,6 +74,17 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
 
     await hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
 
+    @callback
+    def _async_handle_logging_changed(_event: Event) -> None:
+        """Handle when the logging level changes."""
+        manager.debug = logging.getLogger("pyvesync.vesync").level == logging.DEBUG
```

Comment:
Why doesn't the library use the standard Python logging mechanism and best practice with just using a logger per module or package? Why does it need this attribute on the manager?

Reply:
Just how it was written - as webdjoe mentioned a re-write of the library is coming that will solve this.  If we merge this current code it gives us debug ability until that happens.  

It would get removed on the re-write update.